docs/tutorial/security.md

```diff
@@ -250,6 +250,10 @@ see our dedicated [Context Isolation](context-isolation.md) document.
 
 ### 4. Enable process sandboxing
 
+:::info
+This recommendation is the default behavior in Electron since 20.0.0.
```

Comment:
Per the Electron 20 PR that introduced that change (https://github.com/electron/electron/pull/32869):

> Renderers are now sandboxed by default **unless nodeIntegration: true or sandbox: false is specified**.

So I think we should make it clear that sandboxing will _not_ be enabled by default if the user sets `nodeIntegration` to `true`.

Reply:
@erikian Thank you very much for your feedback :)

I'm currently re-reading the documentation and I will submit an updated pull request. The way I understand it, it would also make sense to make small adjustments to _Context isolation_ and _Process sandboxing_. 

I will try to phrase things as clearly as possible and reference the related chapters where necessary.

Reply:
> Should we mention the issue with process isolation in https://www.electronjs.org/docs/latest/tutorial/sandbox as well? There it says in the last paragraph of the opening section: _"Starting from Electron 20, the sandbox is enabled for renderer processes without any further configuration. If you want to disable the sandbox for a process, see [...]"_
> 
> That's where I had gathered my information from, hence the original commit.

Ah fair enough. We do mention the `nodeIntegration` caveat under [Disabling the sandbox for a single process](https://www.electronjs.org/docs/latest/tutorial/sandbox#disabling-the-sandbox-for-a-single-process) ("Sandboxing is also disabled whenever Node.js integration is enabled in the renderer"), but looks like the intro paragraph you quoted is assuming that the default options are used; I think we could add a clarification about the effects of `nodeIntegration: true` to that paragraph as well.

Reply:
I've updated the paragraph. I hope it's concise enough and doesn't add too much clutter.

Under [2 Do not enable Node.js integration for remote content](https://www.electronjs.org/docs/latest/tutorial/security#2-do-not-enable-nodejs-integration-for-remote-content) I've also found another way to enable node integration - I've added that possibility to [Disabling the sandbox for a single process](https://www.electronjs.org/docs/latest/tutorial/sandbox#disabling-the-sandbox-for-a-single-process).

There's also another, seemingly related flag: `nodeIntegrationInWorker: true`. Is that also related to the sandbox feature?

Reply:
> There's also another, seemingly related flag: `nodeIntegrationInWorker: true`. Is that also related to the sandbox feature?

According to https://www.electronjs.org/docs/latest/tutorial/multithreading (there's a link to that page in the docs for `nodeIntegrationInWorker` under [`BrowserWindow`](https://www.electronjs.org/docs/latest/api/browser-window)), it is:

> The `nodeIntegrationInWorker` can be used independent of `nodeIntegration`, but `sandbox` must not be set to `true`.

Reply:
Thank you for digging that up :) 

If I understand it correctly, `sandbox: true` prevents using  `nodeIntegrationInWorker`, not the other way around, as it is with `nodeIntegration`.

That would be perfectly fine from the standpoint of our security recommendation and we wouldn't have to mention it.